packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/delete-database/delete-database.html

```diff
@@ -0,0 +1,12 @@
+<form name="deleteDatabase">
+    <oui-modal
+        data-on-dismiss="$ctrl.goBack()"
+        data-primary-label="{{:: 'pci_databases_databases_delete_database_confirm' | translate}}"
+        data-primary-action="$ctrl.deleteDatabase()"
+        data-secondary-action="$ctrl.cancel()"
```

Comment:
Useless intermediate method
```suggestion
        data-secondary-action="$ctrl.goBack()"
```